docs/sql/settings.md

```diff
@@ -295,3 +295,240 @@ Result set:
   }
 }
 ```
+
+## opendistro.sql.query.response.format
+
+### Description
+
+You can set default response format of the query. The supported formats include: jdbc, json, csv, raw, and table.
+
+1. The default value is jdbc.
+2. This setting is node scope.
+3. This setting can be updated dynamically.
+
+
+### Example 1
+
+You can update the setting with a new value like this.
+
+SQL query:
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
+  "transient" : {
+    "opendistro.sql.query.response.format" : "json"
+  }
+}'
+```
+
+Result set:
+
+```json
+{
+  "acknowledged" : true,
+  "persistent" : { },
+  "transient" : {
+    "opendistro" : {
+      "sql" : {
+        "query" : {
+          "response" : {
+            "format" : "json"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Example 2
+
+After the setting is updated, query result is like:
```

Comment:
"After you update the setting, queries return as JSON."

Reply:
Done.